semgrep/semgrep/cli.py

```diff
@@ -81,6 +82,14 @@ def cli() -> None:
         action="store_true",
     )
     config.add_argument("--precommit", help=argparse.SUPPRESS, action="store_true")
+    config.add_argument(
+        "-j",
+        "--jobs",
```

Comment:
Is `--cpus` more clear?

Reply:
I actually see `-j/--jobs` more commonly used, e.g. `make` and `flake8`